src/features/abstractfeatures.jl

```diff
@@ -0,0 +1,6 @@
+#=
+Abstract types for different types of features
+=#
+
+# Abstract type which subtypes GeneralPairFeature and BondFeature
+abstract type AbstractPairFeature <: AbstractFeature end
```

Comment:
I'm very fine with having these in a separate file...have you tested that the package imports properly with it like this? The reason all the abstract types are in the main ChemistryFeaturization module right now is that when I put them in other files (even if that file was `include`d earlier in the module than other files that depended on it, I kept getting undefined errors. But maybe I was doing something wrong?

Reply:
Generally while defining them in another file, I'd just do the quick and easy thing - including the file in all files that use these structures. But there's probably a more idiomatic and cleaner way to do that in Julia? I'll look into that, and get back to you.

Reply:
yeah that would obviously work, but I also feel like there should be a better way. Also, I think that way leads to a bunch of redefinition warnings during precompilation, which don't hurt anything but are annoying and unsatisfying :P

Reply:
easiest way to figure it out might be to find another example package or two that have a variety of abstract types and see how they handle it

Reply:
Yes, that's what I was planning on doing 😆